HaxeManual/02-types.tex

```diff
@@ -678,12 +678,12 @@ \subsection{Default values}
 \section{Dynamic}
 \label{types-dynamic}
 
-While Haxe has a static type system, this type system can, in effect, be turned off by using the \type{Dynamic} type. A \emph{dynamic value} can be assigned to anything; and anything can be assigned to it. This has several drawbacks:
+While Haxe has a static type system, this type of system can, in effect, be turned off by using the \type{Dynamic} type. A \emph{dynamic value} can be assigned to anything and anything can be assigned to it. This has several drawbacks:
```

Comment:
Dropping the semicolon is ok I suppose, but it is a `type system`. It is not a general property of "this type of systems" to be able to turn them off with a `Dynamic` type.